.github/workflows/android-ci.yml

```diff
@@ -11,55 +11,49 @@ defaults:
     shell: bash -l {0}
 
 jobs:
-  build:
+
+  Build:
+    name: Build
     runs-on: ubuntu-latest
 
     steps:
-    - uses: actions/checkout@v2
-
-    - name: Setup gCloud
-      if: github.event.pull_request.head.repo.full_name == github.repository
-      env:
-        BACKPACK_GKEY: ${{ secrets.BACKPACK_GKEY}}
-        GCLOUD_PROJECT_ID: ${{ secrets.GCLOUD_PROJECT_ID }}
-      run: |
-        gpg --quiet --batch --yes --decrypt --passphrase="$BACKPACK_GKEY" --output $HOME/gcloud-service-key.json gcloud-service-key.json.gpg
-        if [ ! -d "$HOME/google-cloud-sdk/bin" ]; then
-          rm -rf $HOME/google-cloud-sdk
-          export CLOUDSDK_CORE_DISABLE_PROMPTS=1
-          curl https://sdk.cloud.google.com | bash
-        fi
-
-        gcloud version
-        gcloud auth activate-service-account --project $GCLOUD_PROJECT_ID --key-file $HOME/gcloud-service-key.json
-    
-    # We install the NDK as the app requires an older version than bundled in the image
-    - name: Install NDK
-      run: echo "y" | sudo ${ANDROID_HOME}/tools/bin/sdkmanager --install "ndk;20.0.5594570" --sdk_root=${ANDROID_SDK_ROOT}
-
-    - name: nvm install
-      run: nvm install
-
-    - name: npm install
-      run: |
-        nvm use
-        npm ci
-
-    - name: Run tests
-      run: npm test
-
-    - name: Setup JDK 1.8
-      uses: actions/setup-java@v1
-      with:
-        java-version: 1.8
-
-    - name: Gradle Lint
-      run: ./android/gradlew -p android ktlint
-
-    - name: Gradle build
-      run: ./android/gradlew -p android :app:assembleDebug :backpack-react-native:assembleAndroidTest
-
-    # Only run tests if not in a forked repo as it depends on secrets not available to forks
-    - name: Run Android tests
-      if: github.event.pull_request.head.repo.full_name == github.repository
-      run: ./scripts/android/ci-tests.sh
\ No newline at end of file
+      - uses: actions/checkout@v2
+
+      - name: Setup NPM
+        run: |
+          nvm install
+          npm ci
+
+      - name: Validate Gradle Wrapper
+        uses: gradle/wrapper-validation-action@v1
+
+      - name: Gradle Lint
+        run: ./android/gradlew -p android ktlint
+
+      - name: Gradle build
+        run: ./android/gradlew -p android :app:assembleDebug :backpack-react-native:assembleAndroidTest
+
+      - name: Run Android unit tests
+        run: ./android/gradlew -p android :app:Test :backpack-react-native:Test
+  
+  # These tests currently fail so we will need a follow up ticket to fix them.
```

Comment:
Is this how we're leaving them? Is there a follow-up ticket we can reference here?

Reply:
I need to create that, thanks for the reminder! We never actually used to run these tests like this so was a new addition but need Android support to take a look and understand the error!

Reply:
I do however think I might know what the problem is looking at it, so let me take another look first and then if its still an issue will create the appropriate backlog ticket